data_platform/utils/deduplication.py

```
@@ -1,24 +1,58 @@
 from typing import TYPE_CHECKING, Any
 
 if TYPE_CHECKING:
     from data_platform.utils.storage import StorageManager
 
-PRIOR_RUN_POLICIES = frozenset({"prior_runs_same_dataset", "prior_runs_all_datasets"})
+PRIOR_RUN_POLICY = "prior_runs_same_dataset"
+PRIOR_RUN_POLICY_ALIASES = frozenset({"prior_runs_all_datasets"})
+PRIOR_RUN_POLICIES = frozenset({PRIOR_RUN_POLICY}) | PRIOR_RUN_POLICY_ALIASES
 
 
 def policy_includes_prior_runs(policy: Any) -> bool:
```

Ruff (0.16.3)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `policy`

(ANN401)